### PR TITLE
Suggestion for being a bit more user-friendly when boot fails for weird reasons.

### PR DESCRIPTION
**Pull request by xeger**
FYI @fellipec and @mathuin, this is the basic idea (not tested in game due to time constraints).

Mathuin's fixes seem good either way, but I figured I would follow fellipec's habit of prompting the user for a corrective action he might take.

### CODE REVIEW

File: boot/boot.ks [hunk 1]
```diff
@@ -65,43 +100,43 @@ ELSE {
 			REBOOT.
 		}
 		ELSE {
-			PRINT "Career insufficiently advanced to deploy antennas -- aborting reboot!".
+			bootError "Cannot contact KSC. Does your ship have enough fixed antennas?".
```
Comment (mathuin) [inline boot/boot.ks]: Would it be excessive to check for fixed antennas, and only if none are found run the CANDOACTIONS test before trying to deploy antennas?  Right now, I think the `bootError` line will be printed if there are fixed antennas but the VAB is not upgraded.

Comment (xeger) [reply boot/boot.ks]: I thought about it, but the user's required course of action is the same in either case: he needs to Add Moar Antenna, and if he can't do actions, then he can't deploy retractable antennas by himself. Hence the error message we show him, should nudge him to exactly the right course of action. (Let me know if my logic is flawed.)

Comment (mathuin) [reply boot/boot.ks]: No, that makes perfect sense, especially for players who are trying to run the entire mission through the terminal.

Comment (xeger) [reply boot/boot.ks]: @fellipec this may be a good starting point if you end up hacking the boot code on Tuesday. Many thanks for your continued efforts!

**Review by fellipec — CHANGES_REQUESTED** [review]
Function calls need to have its parameter inside parenthesis. 
Running like this raises Unexpected token '"' Errors.

**Comment by xeger** [comment]
Fixed function calls and performed pretty thorough testing of all the "can't-do" paths. (Will assume that can-do continues to work.)

**Pull request merged**